core/index.js

```diff
@@ -1,14 +1,14 @@
 // ## Server Loader
 // Passes options through the boot process to get a server instance back
-var server = require('./server');
+let server = require('./server');
```

Comment:
This is a const.

Reply:
thanks for reviewing @kirrg001  change is in progress..

Reply:
Already done by others.